ipaserver/install/dsinstance.py

```diff
@@ -257,14 +257,15 @@ def __common_setup(self, enable_ssl=False):
 
         self.step("creating directory server user", create_ds_user)
         self.step("creating directory server instance", self.__create_instance)
+        self.step("enabling ldapi", self.__enable_ldapi)
+        self.step("configure autobind for root", self.__root_autobind)
```

Comment:
That would be a nice change, as well as removing `_ldap_mod()` completely, but it's out of scope for this refactoring.